nncf/torch/exporter.py

```diff
@@ -71,7 +71,7 @@ class PTExporter(Exporter):
     This class provides export of the compressed model to the ONNX format.
     """
 
-    _ONNX_DEFAULT_OPSET = 13
+    _ONNX_DEFAULT_OPSET = 14
```

Comment:
@AlexanderDokuchaev do we have some tests for torch export outside pre-commit scope?

Reply:
@ I found PTQ conformance tests with export:
`ImageClassificationTimm` and `ImageClassificationTorchvision` test cases in https://github.com/openvinotoolkit/nncf/blob/develop/tests/post_training/model_scope.py
IMO, It's worth launching it